sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj

```diff
@@ -19,4 +19,8 @@
     <PackageReference Include="System.Linq.Async" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)ForwardsClientCallsAttribute.cs" Link="Shared\Core\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
Add to \eng\Directory.Build.data.targets.

Reply:
This isn't added at that level, all other sdk's add these directly into the proj file
https://github.com/Azure/azure-sdk-for-net/search?l=XML&q=ForwardsClientCallsAttribute

Reply:
sure. For now it's ok. Let's simplify the csproj once we get build.props in order.